app/views/characters/_editor.haml

```diff
@@ -9,9 +9,7 @@
   %td{class: cycle('even', 'odd')}= f.text_field :screenname, placeholder: "Screen Name", class: 'text'
 %tr
   %th.sub Setting
-  %td{class: cycle('even', 'odd')}= f.text_field :setting, placeholder: "Setting", class: 'text'
-  /%td{class: cycle('even', 'odd')}= f.collection_select(:setting_ids,
-  /  Setting.all, :id, :name, {selected: @character.settings.pluck(:id)}, {multiple: true})
+  %td{class: cycle('even', 'odd')}= f.collection_select(:setting_ids, Setting.all, :id, :name, {selected: @character.settings.pluck(:id)}, {multiple: true})
```

Comment:
`@character.settings` not `Setting.all` since we don't need them all loaded and it hits the endpoint?